design-toolkit/components/src/components/search-field/search-field.test.tsx

```diff
@@ -34,4 +32,104 @@ describe('SearchField', () => {
 
     expect(screen.getByPlaceholderText(placeholder)).toBeInTheDocument();
   });
+
+  it('should render with custom classNames', () => {
+    render(
+      <SearchField
+        placeholder='Test'
+        classNames={{
+          searchField: 'custom-search-field',
+          input: 'custom-input',
+        }}
+        aria-label='Test Search Field'
+        data-testid='search-field'
+      />,
+    );
+
+    const searchField = screen.getByTestId('search-field');
+    expect(searchField).toHaveClass('custom-search-field');
+  });
+
+  it('should show loading state', () => {
+    render(
+      <SearchField
+        placeholder='Loading search'
+        isLoading={true}
+        aria-label='Loading Search Field'
+      />,
+    );
+
+    // In loading state, the loading icon should be visible
+    // and the clear button should not be visible
+    expect(screen.queryByRole('button')).not.toBeInTheDocument();
+  });
+
+  it('should render filled variant', () => {
```

Comment:
This test feels sort of meaningless since it's just testing a style, but the actual test is just testing for the placeholder content

Reply:
Fixed.